### PR TITLE
Include socket

### PR DESCRIPTION
**Pull request by jogibear9988**

### CODE REVIEW

**Comment by jogibear9988** [comment]
@AlCalzone this is not the server code. This is the Socket, used by "Connection" to communicate with the server

**Comment by AlCalzone** [comment]
Oh, right. Was confused there for a bit. I'll check it out later.

**Comment by jogibear9988** [comment]
as I now heard from bluefox that the "socket" lib is different for admin, web, and socket adapter I don't think this should be included.
I will remove it from the commit.
Maybe I will create a pull for the admin adapter. but Idroped my hopes that ioBroker will use Typescript in some near feature

**Comment by AlCalzone** [comment]
> I dropped my hopes that ioBroker will use Typescript in some near feature

https://github.com/ioBroker/ioBroker.js-controller/blob/master/packages/common-db/src/index.ts

👆🏻

**Comment by jogibear9988** [comment]
so hopepfully we also switch to es6 and imports for all the libs.  like I've done here.
so should I drop the SocketIo.ts ? And should I commit it to Admin Adapter?

**Comment by AlCalzone** [comment]
Do the last changes mean we should undo the commit to master?

**Comment by jogibear9988** [comment]
@AlCalzone no, the commit to master only is a new build as esm module (additionaly).
the small changes here once should also be merged, but they are not yet finished. 
they are also small fixes and a little bit of documentation.

File: README.md [hunk 1]
```diff
@@ -10,11 +10,24 @@ There are 2 connection types in it:
...
+  await adminConnection.waitForFirstConnection();
+  console.log(await adminConnection.getHosts());
+
 ## Changelog
 <!--
 	Placeholder for the next version (at the beginning of the line):
 	### **WORK IN PROGRESS**
 -->
+### 1.0.10 (2021-12-14)
+* (jogibear998) Add TS Version from socket.io from admin ui
```
Comment (AlCalzone) [inline README.md:29]: That's no longer the correct changelog, right?

**Pull request merged**